.changeset/green-rivers-greet.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/console": patch
+"@wso2is/i18n": patch
```

Comment:
Let's bump myaccount as well

Reply:
Reslved in [cac6419](https://github.com/wso2/identity-apps/pull/4906/commits/cac64194c6613ba352f4d11695dcaaff16ad6283)